ludwig/utils/strings_utils.py

```diff
@@ -29,8 +29,13 @@
 from ludwig.utils.nlp_utils import load_nlp_pipeline, process_text
 
 UNKNOWN_SYMBOL = "<UNK>"
+UNKNOWN_IDX = 3
```

Comment:
nit: An `Enum` for this might be more appropriate.

Reply:
Done.